### PR TITLE
JE-52685 limit installation with low RAM

### PR DESCRIPTION
**Pull request by dzotic91-ukr**

### CODE REVIEW

File: manifest.jps [hunk 1]
```diff
@@ -123,6 +123,8 @@ onUninstall:
 
 onBeforeDelete: delete
 
+onBeforeSetCloudletCount [${targetNodes.nodeGroup}]: stopSetCloudlets
```
Comment (SlavaKatiukha) [inline manifest.jps:126]: check version

File: manifest.jps [hunk 2]
```diff
@@ -179,6 +181,9 @@ actions:
       undeployHook: ${settings.undeployHook}
       undeployHookType: ${settings.undeployHookType}
       withExtIp: ${settings.withExtIp}
+  stopSetCloudlets:
+    - script: return {'result': 'warning', 'message': 'At least 512 MB RAM (4 cloudlets) are recommended for the correct installation of the Let\'s Encrypt add-on.'}
+    - stopEvent
```
Comment (SlavaKatiukha) [inline manifest.jps:186]: invalid. See how to use stopEvent

Comment (dzotic91-ukr) [reply manifest.jps]: done

File: manifest.jps (hunk 1 above)
Comment (dzotic91-ukr) [reply manifest.jps:128]: done

**Pull request closed without merging**